javascript.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+
+function menuChange(e) {
```

Comment:
The popup menu should only appear in the mobile view. However, when I click on nav links in desktop mode, it also opens a popup. Instead, it should take me to the section to which the nav link is attached.
![image](https://user-images.githubusercontent.com/33268927/181159795-60af23df-4479-452a-a422-527be4b9aa62.png)

Reply:
Changed the same.